### DISCUSSION TITLE
How I utilize TeslaMate to make sure my car won't sleep? (How I can increase "Time to Try Sleeping" above 90 min?)

### DISCUSSION THREAD

**Discussion by stas87** [teslamate-org/teslamate#1477] [Q&A]
Hi,

I have my car parked underground where no LTE signal, but has WIFI.
When it goes to sleep it drops the wifi and I cant remotely connect to it.
I dont have sentry mode running.

Can I utilize TeslaMate somehow that it will poll every X minutes to prevent it from going sleep? (It is connected to charging so I dont care about battery)

Thanks

**Comment by LelandSindt** [comment]
sans changes to TeslaMate you could write something that connects to  [MQTT](https://docs.teslamate.org/docs/integrations/mqtt) subscribes to `teslamate/cars/$car_id/geofence` and `teslamate/cars/$car_id/plugged_in` then calls `PUT http://teslamate:4000/api/car/1/logging/resume` at a regular interval.. if `plugged_in == true` and `geofence == "home"`  --- this would keep TeslaMate polling and your car online.
